src/main/java/com/minecolonies/coremod/inventory/InventoryCitizen.java

```diff
@@ -586,136 +302,31 @@ public void markDirty()
      * @param player the player acessing the inventory.
      * @return if the player is allowed to access.
      */
-    @Override
     public boolean isUseableByPlayer(@NotNull final EntityPlayer player)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1161)